flowmachine/flowmachine/features/subscriber/topup_balance.py

```diff
@@ -12,15 +12,14 @@
 
 from ..utilities.sets import EventsTablesUnion
 from .metaclasses import SubscriberFeature
-from flowmachine.utils import standardise_date
+from flowmachine.utils import standardise_date, get_stat
 
 valid_stats = {
```

Comment:
Again, good to update the docstring to reflect that 'median' is not a valid stat here.